tests/integration/defs/kv_cache/test_prefix_aware_scheduling.py

```
@@ -0,0 +1,749 @@
+
+
+# ---------------------------------------------------------------------------
+# Fixtures
+# ---------------------------------------------------------------------------
+
+
+@pytest.fixture(scope="module")
+def ensure_lmbenchmark():
+    """Clone LMBenchmark if it is not already present."""
+    if not os.path.exists(LMBENCHMARK_SCRIPT):
+        subprocess.check_call(["git", "clone", "--depth", "1", LMBENCHMARK_REPO, LMBENCHMARK_DIR])
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)